src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/StaffService.kt

```diff
@@ -30,7 +33,7 @@ class StaffService(
     } ?: throw BusinessException("No POM details found for $prisonNumber")
```

Comment:
I _think_ this line is the cause of the transaction issue we are seeing. Returning a Pair(null, null) instead seems to work. Let's have a catch up to discuss.

Reply:
As discussed, it throws an exception from here which is handled in the calling method and a Pair(null, null) is being returned